docs/admin/catalog.md

```diff
@@ -113,5 +116,39 @@ environment:
   - CONFIG_VARIABLE=${CONFIG_VARIABLE}
 ```
 
+If default values are included, for example:
+
+```yaml
+environment:
+  - CONFIG_VARIABLE=${CONFIG_VARIABLE:-foobar}
+```
+
+The value will be ignored when deploying the Service. For setting default values
```

Comment:
Clarify: The default value (foobar) will be ignored...
To avoid people confusing values and thinking ${CONFIG_VARIABLE} will be ignored.